R/module_teal_data.R

```diff
@@ -48,12 +51,17 @@ ui_teal_data <- function(id, data_module = function(id) NULL) {
   )
 }
 
+ui_teal_data <- ui_teal_data_module
```

Comment:
so inside code this is still `ui_teal_data`?

Reply:
Yes, I made this decision to make it easier to keep the code as is before evaluating how much code should be changed if we replaced ui_teal_data by ui_teal_data_module. I'm checking the code locally and I'm seeing lots of tests failing and I don't want to make it even harder to fix everything (yet).